azext_edge/edge/_help.py

```diff
@@ -1030,13 +1025,14 @@ def load_iotops_help():
                       For OPC UA connector arguments, a value of -1 means that parameter will not be used (ex: --session-reconnect-backoff -1 means that no exponential backoff should be used).
                       A value of 0 means use the fastest practical rate (ex: --default-sampling-int 0 means use the fastest sampling interval possible for the server).
 
-                      For more information on how to create an OPCUA connector, please see aka.ms/opcua_quickstart
+                      For more information on how to create an OPCUA connector, please see aka.ms/opcua-quickstart
```

Comment:
That link is more around quick start using the OPC PLC simulator and creating asset configuration for it.

This is a general-purpose command group for asset endpoint configuration, [this link](https://review.learn.microsoft.com/en-us/azure/iot-operations/discover-manage-assets/overview-manage-assets?branch=pr-en-us-286498) looks more appropriate at the command group level.

The link you have can be used in a specific example tailored to use the simulator, "For more info on asset quick start using OPC PLC simulator please see https://aka.ms/opcua-quickstart"